File: app/src/main/java/closer/vlllage/com/closer/handler/helpers/WindowHandler.java
package closer.vlllage.com.closer.handler.helpers;

import closer.vlllage.com.closer.pool.PoolMember;

public class WindowHandler extends PoolMember {
    public int getStatusBarHeight() {
        int statusBarHeightResId = $(ResourcesHandler.class).getResources().getIdentifier("status_bar_height", "dimen", "android");

        if (statusBarHeightResId == 0) {
            return 0;
        }

        return $(ResourcesHandler.class).getResources().getDimensionPixelSize(statusBarHeightResId);
    }
}

File: app/src/main/java/closer/vlllage/com/closer/handler/helpers/WindowHandler.kt
package closer.vlllage.com.closer.handler.helpers

import com.queatz.on.On

class WindowHandler constructor(private val on: On) {
    val statusBarHeight: Int
        get() {
            val statusBarHeightResId = on<ResourcesHandler>().resources.getIdentifier("status_bar_height", "dimen", "android")

            return if (statusBarHeightResId == 0)
                0
            else on<ResourcesHandler>().resources.getDimensionPixelSize(statusBarHeightResId)

        }
}
